R/QFeatures-missing-data.R

```diff
@@ -183,9 +186,9 @@ setMethod("zeroIsNA", c("QFeatures", "numeric"),
 ##' @rdname QFeatures-missing-data
 setMethod("zeroIsNA", c("QFeatures", "character"),
           function(object, i) {
-              for (ii in i)
-                  object[[ii]] <- zeroIsNA(object[[ii]])
-              object
+              if (any(! i %in% names(object)))
+                  stop("subscript contains invalid names")
```

Comment:
My latest compilation of the `SCP.replication` vignette failed with that message. Did you check `SCP.replication` with these changes?